apps/discover.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/ash
+
+# shellcheck shell=dash
+
+arch=$(uname -m)
+packet_base_url=$(sed -nr 's|.*\bpacket_base_url=(\S+).*|\1|p' /proc/cmdline)
+facility=$(sed -nr 's|.*\bfacility=(\S+).*|\1|p' /proc/cmdline)
+tinkerbell=$(sed -nr 's|.*\btinkerbell=(\S+).*|\1|p' /proc/cmdline | sed -e 's|^http://||' -e 's|/.*||')
+phone_home_url="http://$tinkerbell/phone-home"
+
+discover_address=$(sed -nr 's|.*\bdiscover_address=(\S+).*|\1|p' /proc/cmdline)
+if [ -z "$discover_address" ]; then
+	discover_address="http://tinkerbell.$facility.packet.net"
```

Comment:
Perhaps we could use the `$tinkerbell` variable instead?

Reply:
Also, looking at discover-metal, it seems this should be the exact url to send to. So we might actually want to change this to something like `discover_url` or something along those lines. And with that perhaps it should be `/discover-metal` or `/discover-os` or something along those lines. Thoughts?

Reply:
I agree that `discover_url` is preferable to `discover_address`. And are you saying you think the default url should be `http://tinkerbell.$facility.packet.net/discover-os`? If you confirm that, I'm happy to make that change.

Reply:
Yeah I think so, Would we be able to use the `$tinkerbell` variable instead of using the static default domain?

Reply:
Yep, will do!